Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocListEditableRowControlObject.cs

```diff
@@ -74,6 +76,16 @@ public BocListEditableCellControlObject GetCell (int oneBasedIndex)
       return GetCell().WithIndex(oneBasedIndex);
     }
 
+    /// <summary>
+    /// Gets any validation errors assigned to the row.
+    /// </summary>
+    public BocListValidationError[] GetValidationErrors ()
+    {
+      return Scope.FindAllCss(".bocListDataCellValidationFailureIndicator ul li")
+          .Select(BocListValidationError.Parse)
+          .ToArray();
+    }
+
```

Comment:
`FindAllCss` cannot have a wait condition since it returns an unknown amount of elements so there is no way to know when all elements were found.

I don't think it is necessary to optimize performance here. We are in the webtests so most of the performance goes because of IPC anyway (~5ms per roundtrip overhead). Checking an attribute is as fast as doing the `.FindAllCss` operations with no result. So I don't think it is necessary to optimize here.